database/pgsql/rhelv2_vulnerability.go

```diff
@@ -216,6 +217,11 @@ func (pgSQL *pgSQL) GetRHELv2Vulnerabilities(records []*database.RHELv2Record) (
 
 			addedVulns := uniqueVulns[recordID]
 
+			// Order the vulns in terms of updated first so that if we have multiple of the same vulns
+			// the ones with the latest update time will be used
+			sort.SliceStable(matchedVulns, func(i, j int) bool {
+				return matchedVulns[i].Updated.After(matchedVulns[j].Updated)
```

Comment:
is this safe when updated time may possibly be blank? (if it's possible?)

Reply:
Yes, because these are not pointers and are just time.Time which is zero value safe